en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -613,3 +613,352 @@ In this section, you will be generating an access token using the authorization
+-H 'PSU-IP-Address: 127.0.1.1' \
+-H 'PSU-ID-Type: email' \
+-H 'X-Request-ID: d029c37a-d464-4304-b175-7c27fd7f5728' \
+-H 'Date: Mon, 31 Oct 2022 12:12:08 GMT' \
+-H 'PSU-ID: admin@wso2.com' \
+-H 'Signature: keyId="SN=de730ec5733ead1b,CA=1.2.840.113549.1.9.1=#16116d616c7368616e694077736f322e636f6d,CN=OpenBanking Pre-Production Issuing CA,OU=OpenBanking,O=WSO2 (UK) LIMITED,L=COL,ST=WP,C=LK",algorithm="rsa-sha256", headers="digest x-request-id date psu-id",signature=AEtotwhYvGkGQEZYD0a/rvBwW6JCqJ2hBFha8lQdcSI/zruoxF2cYkeRNSir3pwTYF9QmYwp2aPJZ8LoAozSHNQUs2Q9BmNHVET/g8HFV+7iBzfEqK6Mtyzu0u9u7qhBaLdu3vq1gSyMuQHPFoKgWgTGGVFiL0cTICp1Zka5niCxa6GGGwpwvER9DrApSk3nvwm/+vwHwmEEeP3LoQubvKDyDF6Vj438VLKpMGBl1kWv/WwkLhIUSW6OSVvxX+xWhgYBS0qxEW8FroSJIul1ncbhst8qAgFCmoV4fSon1o9qaEHQwnnIec/qpldbtEOmEZNGVCmnfu2OY1XyVw3Vxw==' \
+-H 'Digest: SHA-256=RBNvo1WzZ4oRRq0W9+hknpT7T8If536DEMBg9hyq/4o=' \
+-H 'TPP-Signature-Certificate: <SIGNATURE_CERTIFICATE>' \
+-H 'Authorization: Bearer <APPLICATION_ACCESS_TOKEN>'
+```
+
+``` tab="Response"
+200 OK
+Header: X-Request-ID: d029c37a-d464-4304-b175-7c27fd7f5728
+{
+   "transactionStatus": "ACCP"
+}
+```
+
+!!! note
+    - This response is constructed by the core banking back end.
+    - The `transactionStatus` property shows the status of the real payment at the time the request arrives at the bank backend.
+    - The response to the request must comply with the response specified in the specification.
+
+### Payment cancellation requests
+
+!!! info
+    This is only available as a WSO2 Update from **WSO2 Open Banking Identity Server Berlin Toolkit Level 1.0.0.5** and
+    **WSO2 Open Banking API Manager Berlin Toolkit Level 1.0.0.5** onwards. For more information on updating, see
+    [Getting WSO2 Updates](../install-and-setup/setting-up-servers.md#getting-wso2-updates).
+
+!!! tip "Before you begin:"
+    1. Update the following files:
+        - `dynamic-endpoint-insequence-1.3.6.xml` insequence file
+        - OpenAPI File
+    2. Redeploy the NextGenPsd2XS2AFramework API.
+
+There are three types of payments as instant payments, bulk payments, and periodic payments.
+
+   - Only bulk payments and periodic payments can be cancelled. 
+   - The bank should decide whether the payment cancellation needs authorization or not.
+
+**DELETE /{payment-service}/{payment-product}/{paymentId}**
+
+To cancel an already submitted payment
+
+- This will go to the core banking back end and respond accordingly.
+
+    !!! note
+        - Successful response code will be either 204 or 202. This is decided by the bank.
+        - If you get 204 as the response code, it indicates that the bank does not need authorization to cancel this payment. Therefore, the payment status must be updated from the banking side as `CANC`. The consent status will be updated as `CANC`.
+        - If you get 202 as the response code, it indicates that the bank needs the authorization to cancel this payment. The consent status will be updated to `ACTC`.
```

Comment:
This status "ACTC" must also be sent from the bank backend.

Reply:
Fixed in 4d634c22b28377dd06f03734b943bf636eac4115.